contrib/puppetlabs-lumogon/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "puppetlabs-lumogon",
+  "version": "0.1.0",
+  "author": "James Fryman",
+  "summary": "Provides a Puppet Type/Provider for Lumogon :: inspecting running containers",
+  "license": "Apache-2.0",
+  "source": "https://github.com/puppetlabs/lumogon.git",
+  "project_page": "https://github.com/puppetlabs/lumogon",
+  "issues_url": "https://github.com/puppetlabs/lumogon/issues",
+  "dependencies": [
+  
+  ],
+  "data_provider": null
+}
```

Comment:
This needs an explicit puppet requirement, e.g.:
https://github.com/puppetlabs/puppetlabs-postgresql/blob/master/metadata.json#L67-L72